__tests__/app.test.js

```diff
@@ -2,9 +2,10 @@ const app = require('../app');
 const request = require('supertest');
 const db = require('../db/connection.js');
 const seed = require('../db/seeds/seed.js');
-const testData = require('../db/data/test-data/index.js');
+const devData = require('../db/data/development-data');
+const endpoints = require('../endpoints.json');
 
-beforeEach(() => seed( testData ));
+beforeEach(() => seed( devData ));
```

Comment:
This was right before! You should **_absolutely_** be seeding with your test data when running your tests!!!